### PR TITLE
[es] Update content/es/docs/reference/glossary/limitrange.md

### PR DESCRIPTION
**Pull request by Karmavil**
/language/es

Description:
Update / fix, extended definition for LimitRange in glossary

Why?:
Because a misconception that I had and I forgot to fix it before the merge

/assign @raelga 
(He dejado información extra como comentario en este mensaje pero no sé si se podrá leer aquí.)

### CODE REVIEW

File: content/es/docs/reference/glossary/limitrange.md [hunk 1]
```diff
@@ -20,4 +20,4 @@ Proporciona restricciones para limitar el consumo de recursos por {{< glossary_t
 
 <!--more-->
 
-LimitRange limita la cantidad de objetos que se pueden crear por su tipo (vease {{< glossary_tooltip text="Workloads" term_id="workload" >}}), así como la cantidad de recursos informáticos que pueden ser requeridos/consumidos por {{< glossary_tooltip text="Pods" term_id="pod" >}} individuales en un espacio de nombres.
+LimitRange limita la cantidad de objetos que se pueden crear por tipo, así como la cantidad de recursos informáticos que pueden ser requeridos/consumidos por {{< glossary_tooltip text="Pods" term_id="pod" >}} o {{< glossary_tooltip text="Contenedores" term_id="container" >}} individuales en un espacio de nombres.
```
Comment (raelga) [inline content/es/docs/reference/glossary/limitrange.md:23]:
```suggestion
LimitRange limita la cantidad de objetos que se pueden crear por tipo, así como la cantidad de recursos informáticos que pueden ser requeridos/consumidos por {{< glossary_tooltip text="Pods" term_id="pod" >}} o {{< glossary_tooltip text="Contenedores" term_id="container" >}} individuales en un {{< glossary_tooltip text="Namespace" term_id="namespace" >}}.
```

**Review by Karmavil — COMMENTED** [review]
I would like to see if **this** (my own review after accept request changes) will trigger the build of the website again.
Edit: It's OK it is building with the right commit

**Review by raelga — APPROVED** [review]
/label tide/merge-method-squash
/approve

**Pull request merged**